features/bootstrap/FeatureContext.php

```diff
@@ -60,13 +60,13 @@ public function __construct(ManagerRegistry $doctrine, Request $request)
     }
 
     /**
-     * Sets the default Accept HTTP header to the JSON-LD mime type.
+     * Sets the default Accept HTTP header to null (workaround to artificially remove it).
      *
      * @BeforeScenario
      */
     public function acceptJsonLd()
     {
-        $this->request->setHttpHeader('Accept', 'application/ld+json');
+        $this->request->setHttpHeader('Accept', null);
```

Comment:
Can we use something more sensible as default and only unset the header when required?